### DISCUSSION TITLE
I  install failed by cargo build for deno v1.21.3

### DISCUSSION THREAD

**Discussion by cuobiezi** [denoland/deno#14600] [Q&A]
**failed info**

failed to add native library /var/folders/j9/_hbrhkj93gq3j04l4543f5580000gn/T/cargo-installPXFWPP/release/gn_out/obj/librusty_v8.a: truncated or malformed archive (offset to next archive member past the end of the archive after member __.SYMDEF)

**question**
brew didn't obtain this version yet . how should I do to update deno ?

Many thanks

**Comment by cjihrig** [comment]
Does running `deno upgrade` work?

**Reply by cuobiezi** [reply]
sudo deno upgrade got "You do not have write permission to '/usr/local/bin/deno'". Thanks

**Reply by cuobiezi** [reply]
I found latest version on asdf.Any way many thanks
